prime-angular-frontend/src/app/shared/components/dialogs/content/claim-enrollee/claim-enrollee.component.ts

```diff
@@ -24,21 +25,41 @@ export class ClaimEnrolleeAction {
 })
 export class ClaimEnrolleeComponent implements OnInit {
   public adjudicators$: BehaviorSubject<Admin[]>;
+  private possibleAction: ClaimActionEnum;
```

Comment:
Feels like this should be `action` since once we have this dialog open it isn't a possibleAction it's the actual action.